src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -243,7 +226,7 @@ public override int Run(AnalyzeOptions analyzeOptions)
             // Because of this, the return code bit for RuleNotApplicableToTarget is not
             // interesting (it will always be set).
 
-            return analyzeOptions.RichReturnCode
+            return analyzeOptions.RichReturnCode != null && analyzeOptions.RichReturnCode.Value
```

Comment:
>analyzeOptions.RichReturnCode != null && analyzeOptions.RichReturnCode.Value [](http://example.com/codeflow?start=19&length=76)

I think this line can be just as 
`analyzeOptions.RichReturnCode == true`

Reply:
Yes, better. I am still resistant to direct comparisons to bool literals but it makes code more concise for nullable vars.